custom/yeksi_naa_reports/sqldata.py

```diff
@@ -764,10 +776,10 @@ def calculate_total_row(self, records):
             if not self.date_in_selected_date_range(record['real_date']):
                 continue
             month_index = self.get_index_of_month_in_selected_data_range(record['real_date'])
-            if self.denominator_exists(record['count_products_select']):
+            if record['count_products_select']:
                 if record['nb_products_stockout']:
-                    data[month_index]['nb_products_stockout'] += record['nb_products_stockout']['html']
-                data[month_index]['count_products_select'] += record['count_products_select']['html']
+                    data[month_index]['nb_products_stockout'] += record['nb_products_stockout']
+                data[month_index]['count_products_select'] += record['count_products_select']
```

Comment:
for my own education what was the change that caused these `html` indexes to no longer be necessary? i never fully understood the data structure of the thing producing these records, but didn't see where that changed.

Reply:
Change in columns to simple column rather than sum. Sum created not wanted results. While being provided with 3 records for same place and time out of which numerators were let's say 3, 3 and 3, as well as denominators of values None, 0, 13. It produced 9/13 rather than 3/13.

Reply:
got it. now I'm curious why the numerators were higher than the denominators :upside_down_face: